private/buf/cmd/buf/command/curl/curl.go

```diff
@@ -202,6 +210,9 @@ type flags struct {
 
 	// Handling request and response data and metadata
 	UserAgent string
+	UserCreds string
```

Comment:
nit: almost all the variables in `flags` across our commands are named after the flag in practice, so just name this `User`.

Reply:
done